src/widgets/net.rs

```diff
@@ -139,10 +145,17 @@ impl Widget for NetWidget<'_, '_> {
 		buf.set_string(
 			top_half.x + 1,
 			top_half.y + 2,
-			format!(
-				"Rx/s:     {}/s",
-				Size::Bytes(self.bytes_recv.last().unwrap().to_owned())
-			),
+			if self.show_bits {
+				format!(
+					"Rx/s:     {} bits/s",
```

Comment:
Everything looks good except we need the 'bits' label to change to 'Kb', 'Mb', etc as the amount grows. Maybe there's a rust library that can do this for us or we can write our own function to do this.